be/src/services/persistence-service.ts

```diff
@@ -50,4 +54,4 @@ class PersistenceService {
     }
 }
 
-export default new PersistenceService();
+export const instance =  new PersistenceService();
```

Comment:
Why the change? Was PersistenceService() called multiple times, so 
https://nodejs.org/api/modules.html#modules_caching
does not work here? I just want to know this for the future

Reply:
We're distinguishing between declaration (PersistenceService) and (singleton) instance (`instance`).

If you look at the `VocabularyService`, there we have the `PersistenceService` as field-parameter to avoid the repeated static access to the instance.

The weird import `instance as PersistenceService` is just used in the `HealthRouter` to keep the impact minimal:
You could remove the `as PersistenceService` part if you replace all `PersistenceService` with `instance` in the code.